chcemvediet/apps/inforequests/admin.py

```diff
@@ -354,19 +354,19 @@ def render_delete_form(self, request, context):
         return super(ActionAdmin, self).render_delete_form(request, context)
 
     def delete_model(self, request, obj):
-        outbound, inbound = self.collect_nested_inforequestemail(obj)
+        outbound, inbound = self.nested_inforequestemail_queryset(obj)
         outbound.delete()
         inbound.update(type=InforequestEmail.TYPES.UNDECIDED)
         super(ActionAdmin, self).delete_model(request, obj)
 
-    def collect_nested_inforequestemail(self, action):
+    def nested_inforequestemail_queryset(self, action):
```

Comment:
@viliambalaz FIY: Premenoval som tu metodu, nech z nazvu vyplyva, ze to vracia querysety